### ISSUE TITLE
Debug messages are appearing

### ISSUE THREAD

**Issue by pnemade** [flathub/org.gnome.Evince#20]
```
> $ flatpak run org.gnome.Evince 1.pdf
> 
> (evince:2): Gdk-WARNING **: 22:00:25.330: Failed to read portal settings: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such interface “org.freedesktop.portal.Settings” on object at path /org/freedesktop/portal/desktop
> Error registering document: GDBus.Error:org.freedesktop.systemd1.NoSuchUnit: Unit org.gnome.Evince.service not found.
> 
```

so I just ran a evince flatpak application with some local file 1.pdf and above is the result. Can this flatpak be fixed to not give above messages?

**Comment by pnemade** [comment]
> Which version are you running?

> $ flatpak info org.gnome.Evince
Ref: app/org.gnome.Evince/x86_64/stable
ID: org.gnome.Evince
Arch: x86_64
Branch: stable
Origin: flathub
Collection ID: org.flathub.Stable
Date: 2018-11-18 17:41:04 +0000
Subject: Update poppler to 0.71.0 (dc2d491a)
Commit: 80ec831b2afc4c9383e2c8b0ddacc9b47a99075acaf2a397167900c26d5af76f
Parent: 8040fa11e1b16b04819489e6930aa75de56881de48c4d6e26f4dd01380dd0832
Location: /var/lib/flatpak/app/org.gnome.Evince/x86_64/stable/80ec831b2afc4c9383e2c8b0ddacc9b47a99075acaf2a397167900c26d5af76f
Installed size: 28.0 MB
Runtime: org.gnome.Platform/x86_64/3.30
Sdk: org.gnome.Sdk/x86_64/3.30

**Comment by hadess** [comment]
> > (evince:2): Gdk-WARNING **: 22:00:25.330: Failed to read portal settings: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such interface “org.freedesktop.portal.Settings” on object at path /org/freedesktop/portal/desktop

This is generated by the GTK+ in the sandbox, not by Evince.

> > Error registering document: GDBus.Error:org.freedesktop.systemd1.NoSuchUnit: Unit org.gnome.Evince.service not found.

This is a repeat of: https://github.com/flathub/org.gnome.Evince/issues/12

**Comment by pnemade** [comment]
Thank you @hadess for your quick and informative reply. I need one more thing to understand here.
I saw from above comment that systemd unit error is fixed in nightly builds. Hence, I installed the gnome-nightly-app evince as well. Now when I ran the same command as

> flatpak run org.gnome.Evince//stable 1.pdf

This did not produce that systemd unit error. So I removed nightly Evince flatpak and ran the above command again and can reproduce the systemd unit error.

I did not understand what makes that error not to appear. I only installed nightly evince app but ran stable evince app.
